botmain.js

```diff
@@ -1,132 +1,154 @@
-const { Client, GatewayIntentBits, Partials, Collection, EmbedBuilder, Events, ActionRowBuilder, StringSelectMenuBuilder, MessageEmbed } = require('discord.js');
-const config = require('./environmentConfig')
-const timetableBuilder  = require('./timetable/timetableUtils');
-const timetableData = require('./timetable/timetables.json');
-const Classes = require('./timetable/timetables.json');
-const TxtEasterEgg = require('./functions/TxtEasterEgg.js');
-const dashboard = require('./functions/dashboard.js');
-const dotenv = require('dotenv');
-const path = require('path');
-const fs = require('fs');
-const cron = require('node-cron');
-require('date-utils');
-const {configPath} = require("./environmentConfig");
+const {
+  Client,
+  GatewayIntentBits,
+  Partials,
+  Collection,
+  EmbedBuilder,
+  Events,
+  ActionRowBuilder,
+  StringSelectMenuBuilder,
+  MessageEmbed,
+  ActivityType,
+} = require("discord.js");
```

Comment:
ここも1行の方がいい
複数行だとコード長くなるだけで見にくい
```suggestion
const {Client, GatewayIntentBits, Partials, Collection, EmbedBuilder, Events, ActionRowBuilder, StringSelectMenuBuilder, MessageEmbed, ActivityType} = require("discord.js");
```